CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support for the `v` prefixed version
```

Comment:
Can you add more detail on why we added this. So we can support using the flux image automation controller with App CRs.

Reply:
done